src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -355,10 +357,26 @@ private static bool InSymbolHeader(SyntaxNode matchingNode, int position)
                 return parenLambda.WithParameterList(parenLambda.ParameterList.WithParameters(updatedParameters));
             }
 
+            // Handle references in crefs
+            if (updatedNode.IsKind(SyntaxKind.NameMemberCref, out NameMemberCrefSyntax? nameMemberCref))
```

Comment:
This if-check was just being moved from farther down the file since it doesn't need the semantic model that the rest of the cases do.